topics/galaxy-interface/tutorials/workflow-posters/tutorial.md

```diff
@@ -0,0 +1,271 @@
+---
+layout: tutorial_hands_on
+
+title: Creating high resolution images of Galaxy Workflows
+tags:
+- workflows
+questions:
+- How can I print my Workflow in high resolution?
+- How can I share high detail images of my Workflows?
```

Comment:
```suggestion
- How can I print or screenshot my Workflow in high resolution for a poster or presentation?
- How can I share high detail images of my Workflows?
```

(i suspect some users will be looking for 'screenshot' as a keyword, and poster is probably the №1 reason they will use this feature.)